app/views/school_groups/recent_usage.html.erb

```diff
@@ -1 +1,49 @@
 <%= render 'enhanced_header' %>
+
+<table width="100%" class="mt-3 table table-borderless table-sorted school-groups-recent-usage-table">
```

Comment:
Can you add the advice-table class so we end up with a table that matches the styling on the advice pages/school dashboard?